### PR TITLE
Add queue module per ticket request #560

### PR DESCRIPTION
**Pull request by davidjeddy**

### CODE REVIEW

**Comment by trntv** [comment]
What about listeners?

**Comment by davidjeddy** [comment]
Source module has more detailed information on how to implement the finer points of the queue. I did not want to copy/paste the entire readme into the starter kit readme. From source:
```markdown
Command launches a daemon which infinitely queries the queue:

yii queue/listen
```

File: common/config/base.php [hunk 1]
```diff
@@ -147,7 +147,12 @@
                 'baseUrl' => env('STORAGE_BASE_URL'),
             ],
             require(Yii::getAlias('@storage/config/_urlManager.php'))
-        )
+        ),
+
+        'queue' => [
+            'class' => \yii\queue\file\Queue::class,
+            'path' => '@runtime/queue',
```
Comment (trntv) [inline common/config/base.php]: pls, change to
```
@common/runtime/queue
```

``@runtime`` is relative to current application eg frontend, backend, console

File: README.md [hunk 2]
```diff
@@ -304,6 +304,47 @@ In view:
...
+
+Pushes job into queue that run after 5 min:
+
+```php
+Yii::$app->queue->delay(5 * 60)->push(new DownloadJob([
+    'url' => 'http://example.com/image.jpg',
+    'file' => '/tmp/image.jpg',
+]));
+```
+
```
Comment (trntv) [inline README.md]: can you add a few words about how to start listeners? it doubtless that we'll have questions about it ;-)

Comment (davidjeddy) [reply README.md]: Added below.

**Comment by trntv** [comment]
thank you

**Pull request merged**